### PR TITLE
Adding fast-check to the shrinking challenge

### PR DESCRIPTION
**Pull request by dubzzz**
@jlink First of all, thanks a lot for this challenge. As promised a while ago, here is a new property based testing to enrich your challenge with more examples.

Please let me know if I should adapt things in the current PR.

Related to https://github.com/dubzzz/fast-check/issues/899

### CODE REVIEW

**Comment by dubzzz** [comment]
Still a WIP, I have some wrongly written properties

File: README.md [hunk 1]
```diff
@@ -55,22 +55,23 @@ Some challenges were stolen from
...
+| Challenge                                           | Hypothesis                                                         | jqwik                                                      | PropEr                                                        | FsCheck                                                       | fast-check                                                      |
+| --------------------------------------------------- | ------------------------------------------------------------------ | ---------------------------------------------------------- | ------------------------------------------------------------- | ------------------------------------------------------------- | --------------------------------------------------------------- |
+| [bound5](/challenges/bound5.md)                     | [report](/pbt-libraries/hypothesis/challenges/bound5.md)           | [report](/pbt-libraries/jqwik/reports/bound5.md)           |                                                               | [report](/pbt-libraries/fscheck/challenges/Bound5.md)         | [report](/pbt-libraries/fast-check/reports/bound5.md)           |
```
Comment (dubzzz) [inline README.md]:
```suggestion
| [bound5](/challenges/bound5.md)                     | [report](/pbt-libraries/hypothesis/challenges/bound5.md)           | [report](/pbt-libraries/jqwik/reports/bound5.md)           |                                                               | [report](/pbt-libraries/fscheck/challenges/bound5.md)         | [report](/pbt-libraries/fast-check/reports/bound5.md)           |
```

**Comment by jlink** [comment]
@dubzzz Sorry. I somehow missed the PR. Is it ready for merging from your POV?

**Comment by dubzzz** [comment]
Yes, ready to be merged

**Comment by dubzzz** [comment]
I'll certainly update the results with future releases of fast-check (mostly the calculator one where fast-check was not really efficient with current version).

But it can be merged for now

**Comment by jlink** [comment]
Many thanks for the great contribution, Nicolas!

Am Do., 28. Jan. 2021 um 19:57 Uhr schrieb Nicolas DUBIEN <
notifications@github.com>:

**Pull request merged**